scripts/reset_default.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
+
+import pandas as pd
+from cashctrl_ledger import CashCtrlLedger
+from pyledger import StandaloneLedger
+
+def main():
+    cashctrl_ledger = CashCtrlLedger()
+
+    # Delete journals
+    journals = cashctrl_ledger._client.list_journal_entries()
+    ids = ','.join(journals['id'].astype(str).tolist())
+    if len(ids):
+        cashctrl_ledger._client.post("journal/delete.json", {'ids': ids})
+
+    # Restore default VAT with delete=False
+    initial_vat = pd.read_csv('scripts/initial_vat.csv')
```

Comment:
Accessing a relative file path hinders portability. The script can not be run from any location.

I suggest to hard-code initial VAT codes instead.
(An alternative way to make the data location-independent would be to include it in the package. Seems overly complicated for our case).

Here's an example how a hard coded csv file can be added:
https://github.com/macxred/cashctrl_ledger/issues/25#issuecomment-2138098295